temporian/core/operators/BUILD

```diff
@@ -1,23 +1,26 @@
 package(
-    default_visibility = ["//visibility:public"],
-    licenses = ["notice"],
+    default_visibility=["//visibility:public"],
```

Comment:
This is a strange formatting. BUILD files are generally treated like python files.
Can you check?

Reply:
@DonBraulio if you're using vscode, you can install the bazel extension to have automatic formatting on `BUILD` files